doc/usage/configuration.rst

```diff
@@ -2683,6 +2683,19 @@ Options for the linkcheck builder
 
    .. versionadded:: 3.4
 
+.. confval:: linkcheck_exclude_documents
+
+   A list of regular expressions that match documents in which Sphinx should
+   not check the validity of links. This can e.g. be used for permitting link
+   decay in legacy or historical sections of the documentation.
```

Comment:
Reads better without the `e.g.`. Up to you.
```suggestion
   not check the validity of links. This can be used for permitting link decay
   in legacy or historical sections of the documentation.
```